### PR TITLE
Drop py2.6 support from travis.

### PR DESCRIPTION
**Pull request by tomprince**
This is for testing travis.

### CODE REVIEW

**Comment by tardyp** [comment]
not sure exactly why you want to hold this. 👍 please merge when you are ready

**Comment by tomprince** [comment]
At least
- documentation
- updating metabbot to also drop the configuration.

File: .travis.yml [hunk 1]
```diff
@@ -54,42 +73,14 @@ cache:
...
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=coverage
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=lint
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=js
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=docs
-
+    - python: "2.6"
+      env: TWISTED=14.0.2 TESTS=worker-slave
```
Comment (rutsky) [inline .travis.yml]: I suggest to use just `worker` here for indentifier.

Comment (rutsky) [inline .travis.yml]: On the second though, I think better would be `workers`. And instead of `trial_worker` use `only_worker`.

Comment (rutsky) [inline .travis.yml]: In accordance with bug 2340 I would like to reduce usage of "slave" as much, as possible.

**Comment by tomprince** [comment]
https://github.com/buildbot/metabbotcfg/pull/40 should be merged and deployed before this lands.

File: .travis.yml [hunk 2]
```diff
@@ -54,44 +75,14 @@ cache:
...
-      env: TWISTED=15.5.0 SQLALCHEMY=latest TESTS=trial
-
-    # add extra tests in separate jobs
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=js
-
+    - python: "2.6"
+      env: TWISTED=14.0.2 TESTS=trial_worker
+    - python: "2.6"
+      env: TWISTED=15.4.0 TESTS=trial_worker
```
Comment (rutsky) [inline .travis.yml]: So effectively `trial_worker` is run of `trial buildslave.test buildbot_worker.test` on Python 2.6?

Comment (tomprince) [inline .travis.yml]: Yes.

Comment (rutsky) [inline .travis.yml]: This non-obvious to me from `trial_worker` name, but explains why:

1. `TWISTED=latest` is not specified for it (latest Twisted doesn't support Python 2.6).
2. `master` is not installed in this configuration (Python-level master-worker communication, i.e. local worker, works only with buildbot-worker on Python 2.7, which is tested in `TESTS=trial` configuration).

Maybe we should add comment regarding this?

**Comment by rutsky** [comment]
With the exception of non-obvious `trial_worker` configuration behavior, changes LGTM.

**Pull request merged**